migrations/versions/a41fc890a5a8_.py

```diff
@@ -0,0 +1,58 @@
+"""empty message
+
+Revision ID: a41fc890a5a8
+Revises: 3dbfe879d1da
+Create Date: 2021-02-15 13:09:22.391192
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'a41fc890a5a8'
+down_revision = '3dbfe879d1da'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('performance_profile',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('inventory_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('performance_record', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
+    sa.Column('performance_score', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
+    sa.Column('report_date', sa.Date(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('inventory_id', 'report_date')
+    )
+    op.create_table('rh_accounts',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.Text(), nullable=False),
+    sa.CheckConstraint('NOT(name IS NULL)'),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('systems',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('account_id', sa.Integer(), nullable=True),
+    sa.Column('inventory_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('inventory_id')
+    )
+    op.create_table('reports',
+    sa.Column('system_id', sa.Integer(), nullable=False),
+    sa.Column('reported', sa.Date(), nullable=False),
```

Comment:
We can just alter the performance_profile table to contain a foreign key into the `systems` table.

Reply:
done.